lib/fx/version.rb

```diff
@@ -1,4 +1,4 @@
 module Fx
   # @api private
-  VERSION = "0.7.0"
+  VERSION = "1.7.1"
```

Comment:
You don't need to change this, I'll do that once a release is ready.

```suggestion
  VERSION = "0.7.0"
```

Or to be on top of master

```suggestion
  VERSION = "0.8.0"
```